label_studio_ml/examples/segment_anything_model/docker-compose.yml

```diff
@@ -1,29 +1,34 @@
 version: "3.8"
 
 services:
-  redis:
-    image: redis:alpine
-    container_name: redis
-    hostname: redis
-    volumes:
-      - "./data/redis:/data"
-    expose:
-      - 6379
   server:
     container_name: server
-    build: .
+    build:
+      context: .
+      shm_size: '4gb'
+    deploy:
+      resources:
+        limits:
+            memory: 8G
+        reservations:
+            memory: 4G
     environment:
+      # Change this to your model name
+      - SAM_CHOICE=MobileSAM
+      - VITH_CHECKPOINT=/app/sam_vit_h_4b8939.pth
+      - MOBILESAM_CHECKPOINT=/app/mobile_sam.pth
+      - ONNX_CHECKPOINT=/app/sam_onnx_quantized_example.onnx
       - MODEL_DIR=/data/models
-      - RQ_QUEUE_NAME=default
-      - REDIS_HOST=redis
-      - REDIS_PORT=6379
-      - LABEL_STUDIO_USE_REDIS=true
+      - LOG_LEVEL=DEBUG
+      # Add these variables if you want to access the images stored in Label Studio
+      - LABEL_STUDIO_HOST=
+      - LABEL_STUDIO_ACCESS_TOKEN=
     ports:
       - 9090:9090
-    depends_on:
-      - redis
-    links:
-      - redis
     volumes:
       - "./data/server:/data"
```

Comment:
I think for the docker-compose, we should instead assume the weights are loaded into the Dockerfile. While this implementation is more flexible, to get users started as quickly as possible in a wide variety of different situations, I think we should strive to have an all-in-one solution.

Reply:
Same here, I meant to highlight the docker-compose volume mounts for the weights.

Reply:
I've included MobileSAM inside the docker